packages/amplify-category-function/src/provider-utils/awscloudformation/utils/getDependentFunction.ts

```diff
@@ -29,12 +31,26 @@ export async function lambdasWithApiDependency(
 
     if (typeof selectedCategories === 'object' && selectedCategories !== null) {
       for (const selectedResources of Object.values(selectedCategories)) {
-        deletedModelFound = Object.keys(selectedResources).some(r => modelsDeleted.includes(r));
+        deletedModelFound = Object.keys(selectedResources).some(r => !(existingModels.includes(r)));
         if (deletedModelFound) {
           dependentFunctions.push(lambda);
         }
       }
     }
+    else {
+      // In the case that the function-parameters.json does not have info on dependencies, we check the CFN stack.
+      // If the dependency is only a dynamo stream trigger, we don't have info in function-parameters.json to work with
+      const lambdaStackJson = loadFunctionStackAsJSON(resourceDirPath, lambda.resourceName);
+      if(lambdaStackJson?.Resources) {
```

Comment:
It doesn't look like the linter was run on these changes.

Reply:
The linter isn't giving me any concerns with these lines, is there something specific that looks stylistically wrong?

Reply:
Prettier should add a space in the if statement: `if (...)`

Reply:
Yea, what @jhockett said. I'm not sure what's going on with your local setup, but it keeps letting these style issues creep up in PRs.